src/ExtensionsSample/Samples/SendGridSamples.cs

```diff
@@ -20,32 +19,63 @@ public static class SendGridSamples
                 To = "{CustomerEmail}",
                 Subject = "Thanks for your order (#{OrderId})!",
                 Text = "{CustomerName}, we've received your order ({OrderId}) and have begun processing it!")]
-            SendGridMessage message)
+            out SendGridMessage message)
```

Comment:
This is a breaking change over the first release of this extension. However our next release we're revving our major version, so this is OK - all the functionality remains if you change to out.